src/node_sea.cc

```diff
@@ -113,13 +79,13 @@ void Initialize(Local<Object> target,
                 Local<Value> unused,
                 Local<Context> context,
                 void* priv) {
-  SetMethod(
-      context, target, "getSingleExecutableCode", GetSingleExecutableCode);
+  Environment* env = Environment::GetCurrent(context);
+  Isolate* isolate = env->isolate();
+  READONLY_PROPERTY(
```

Comment:
Doesn't this end up in the snapshot? Does SEA still works after this?

Reply:
Oh I guess it works for the default build, which doesn't run the custom entry point...but we should probably somehow assert that SEA is in conflict with snapshot for now (until something like https://github.com/nodejs/node/pull/46824 is supported).

Reply:
Or we can just make `isSea` a property getter and in `lib/internal/main/embedding.js` assert that `v8.isBuildingSnapshot()` is false when `isSea` is true.

Reply:
Turned this into a function in a0e1149ca86a3259381fbfc3a8186960f431e9a8 :+1: